.github/workflows/test-shared.yml

```diff
@@ -0,0 +1,21 @@
+name: Test shared
+
+on:
+    workflow_dispatch:
+    pull_request:
+    push:
+        branches: [main]
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-node@v4.0.0
+              with:
+                  node-version: 22
+                  cache: 'npm'
+            - run: npm ci
+            - run: npm run check
+            - run: npm test
+            - run: npm pack
```

Comment:
Is packing necessary here?

Reply:
It is not really necessary, but I try to do as many steps towards releasing without actually releasing to check whether anything might go wrong _if_ we want to release.